app/helpers/object_link_helper.rb

```diff
@@ -140,20 +140,26 @@ def description_link(desc)
     link_with_query(result, desc.show_link_args)
   end
 
+  # Array of links to searches on external sites;
```

Comment:
@mc-nathan: I hope this is what you had in mind when suggesting constant dictionary plus small helper method.

Reply:
Looks good!  As a really minor point, I'm used to seeing constants like LOCATION_SEARCH_URLS defined before they get used.

Reply:
Moved. Thanks for the direction here and in response to my other question about location of constants.